.editorconfig

```diff
@@ -0,0 +1,39 @@
+# https://EditorConfig.org
+
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
+
+[*.ipynb]
+# Content is json, but it seems to be minimally formatted
+indent_size = unset
+
+[{*.json,.ecrc}]
+indent_size = 2
+
+[Makefile]
+indent_style = tab
+
+[make.bat]
+end_of_line = crlf
+indent_style = tab
```

Comment:
Do we currently have any Makefiles?
I would keep this simple and only add once needed